rfd/0036/user-stories/jupiter-example-com-jenkins.md

```diff
@@ -0,0 +1,77 @@
+# Building and testing jupiter.example.com with Jenkins
+
+The trivial jupiter.example.com application consists of a number of stateless containers (or containers whose state can be easily reconstructed) running as a tiered application of Nginx fronting a PHP/Apache web content templating system.
+
+### Jenkins
+
+The operators use Jenkins to automate the process of building new containers for each pull request and branch in the origin repo. This requires that Jenkins jobs need to be able to interact with Mariposa via the CLI (or API).
+
+- Jenkins must be provisioned with an SSH key for Mariposa
+- Jenkins must be provisioned with an SSH key for GitHub deploy keys
+- Jenkins will effectively own the entire account until we have RBAC v2.
```

Comment:
This is a good callout, but it doesn't help us specify what permissions would be expected in an RBACv2 world